react/getting_started_with_react/react_components.md

```diff
@@ -37,7 +37,7 @@ function Greeting() {
 
 This might look mostly familiar to you - it's a simple JavaScript function, which returns JSX. Open up the project you were working on, create a new file named `Greeting.jsx`, and in that file write your own handmade functional component. Name it whatever you wish, and have it return whatever JSX you wish. 
 
-Are you done? Check the naming of your function! Is it capitalized? Keep this key difference in mind, components that aren't capitalized might not run as expected - which is why we capitalized `Greeting()`.
+Are you done? Check the naming of your function! Is it capitalized? Keep this key difference in mind. **React components must be capitalized** or they may not function as expected, which is why we capitalized `Greeting()`.
```

Comment:
```suggestion
Are you done? Check the naming of your function! Is it capitalized? Keep this key difference in mind. **React components must be capitalized** or they may not function as expected, which is why we capitalized `Greeting()`.
```

Should we perhaps add why this happens somewhere in the lesson? A note about JSX parsing and its recognition as React components vs. regular HTML components/elements?